tools/ImportExamples.ps1

```diff
@@ -358,15 +358,24 @@ function Update-ExampleFile {
 
 
         }else{
-            if(-not($Content | Select-String -pattern $CommandPattern)){
             Clear-Content $ExampleFile -Force
+            if($Content | Select-String -pattern $CommandPattern){
+                Retain-ExistingCorrectExamples -Content $Content -File $ExampleFile
+            }else{
             #Replace everything with boiler plate code
-            $DefaultBoilerPlate = "### Example 1: {{ Add title here }}`r`n``````powershell`r`n PS C:\> {{ Add code here }}`r`n`n{{ Add output here }}`r`n```````n`n{{ Add description here }}`r`n`n### Example 2: {{ Add title here }}`r`n``````powershell`r`n PS C:\> {{ Add code here }}`r`n`n{{ Add output here }}`r`n```````n`n{{ Add description here }}`r`n`n"
-            Add-Content -Path $ExampleFile -Value $DefaultBoilerPlate.Trim()
+                $DefaultBoilerPlate = "### Example 1: {{ Add title here }}`r`n``````powershell`r`n PS C:\> {{ Add code here }}`r`n`n{{ Add output here }}`r`n```````n`n{{ Add description here }}`r`n`n### Example 2: {{ Add title here }}`r`n``````powershell`r`n PS C:\> {{ Add code here }}`r`n`n{{ Add output here }}`r`n```````n`n{{ Add description here }}`r`n`n"
```

Comment:
It looks like this logic of removing example placeholders isn't working as per this change in your PR - https://github.com/microsoftgraph/msgraph-sdk-powershell/blob/80dcb83c163b428f266fe8988d0851771fc90b38/src/Calendar/beta/examples/New-MgBetaUserEventExtension.md.

You can for example consider a generic implementation like:
- Trim whitespaces from both ends of the string content (md file with placeholder text).
- Find and replace instances where the string content starts with `Example .*: {{ Add title here }}` and ends in `{{ Add description here }}`.

Reply:
The objective of that particular block is not to remove the boiler plate code. It instead sets it as a content to the md file incase the file contains nothing that matches the command name and also if whatever the content we are getting from the api reference does not match the command name.  It therefore corrects any historical errors in regards to files that have examples not related to the command names.

Reply:
@timayabi2020, what's the effect of leaving the example blank instead? We shouldn't be showing boiler plate text when an example doesn't exist in the cmdlet reference docs. Are removing the boiler plate text in the SDK docs repro by any chance? If not, please create an issue to track this.

Reply:
Yes I am removing them. However, I will create another issue to transfer that functionality from the reference docs pipeline to this example import script